terraform/modules/openstack/main.tf

```diff
@@ -313,6 +313,19 @@ resource "openstack_compute_secgroup_v2" "nfs-server" {
   }
 }
 
+resource "openstack_compute_secgroup_v2" "iRods" {
```

Comment:
Non-standard name (lowercase only).

Reply:
thanks, will change to lowercase everywhere